### ISSUE TITLE
Deprecation warning when using with webpack5

### ISSUE THREAD

**Issue by Eliasvdb** [sn-satyendra/webpack-font-preload-plugin#40]
Hi

I noticed this error after I started using the webpack-font-preload-plugin with webpack5:

```
(node:30819) [DEP_WEBPACK_COMPILATION_ASSETS] DeprecationWarning: Compilation.assets will be frozen in future, all modifications are deprecated.
BREAKING CHANGE: No more changes should happen to Compilation.assets after sealing the Compilation.
        Do changes to assets earlier, e. g. in Compilation.hooks.processAssets.
        Make sure to select an appropriate stage from Compilation.PROCESS_ASSETS_STAGE_*.
    at WebpackFontPreloadPlugin.addFonts ((...)/node_modules/webpack-font-preload-plugin/src/index.js:76:40)
    at (...)/node_modules/webpack-font-preload-plugin/src/index.js:37:39
    at Hook.eval [as callAsync] (eval at create ((...)/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:17:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] ((...)/node_modules/webpack/node_modules/tapable/lib/Hook.js:18:14)
    at Compiler.emitAssets ((...)/node_modules/webpack/lib/Compiler.js:873:19)
    at (...)/node_modules/webpack/lib/Compiler.js:461:10
    at processTicksAndRejections (internal/process/task_queues.js:77:11)
```

Is this a known issue?

**Comment by sn-satyendra** [comment]
@Eliasvdb yes this is a known issue https://github.com/sn-satyendra/webpack-font-preload-plugin/issues/32 and hopefully will be addressed in the next release (2.0.0).

**Comment by Eliasvdb** [comment]
Oh I missed that, my bad, thanks!
